apps/explorer/src/pages/transaction-result/SendReceiveView.tsx

```diff
@@ -1,56 +1,180 @@
+                                <div className={styles.oneaddress}>
+                                    <div className={styles.doneicon}>
+                                        <DoneIcon />
+                                    </div>
+                                    <Longtext
+                                        text={add}
+                                        category="addresses"
+                                        isLink={true}
+                                        alttext={add}
+                                    />
+                                </div>
+                                <div className={styles.sui}>
+                                    <span className={styles.suiamount}>
+                                        {amount?.[idx].toString()}
+                                    </span>
+                                    <span className={styles.suilabel}>
+                                        {coinList.loadState === 'loaded'
+                                            ? coinList.data[idx]
+                                            : ''}
+                                    </span>
+                                </div>
+                            </div>
+                        ))}
+                    </div>
+                )}
+            </div>
+        </div>
+    );
+}
+
+//TODO: Add date format function
+function SendReceiveView({ sender, recipient, amount, objects }: TxAddress) {
+    const [isShortScreen, setIsShortScreen] = useState(false);
+
+    const { innerWidth } = window;
+
+    useEffect(() => {
+        setIsShortScreen(innerWidth < 440);
+    }, [innerWidth]);
+
+    if (recipient && recipient.length === 1) {
+        return (
+            <div className={styles.txaddress}>
+                <h4 className={styles.oneheading}>Sender &#x26; Recipient</h4>
+                <div className={cl([styles.oneaddress, styles.senderwline])}>
+                    <div className="z-0">
+                        <StartIcon />
+                    </div>
+                    <Longtext
+                        text={sender}
+                        category="addresses"
+                        isLink={true}
+                    />
+                </div>
+                <div>
+                    {recipient.map((add: string, idx: number) => (
+                        <div key={idx} className="flex">
+                            <div
+                                className={cl([
+                                    styles.oneaddress,
+                                    'mt-[20px] ml-[10px] w-[90%]',
+                                ])}
+                            >
+                                <div
+                                    className={`${styles.doneicon} ${
+                                        styles.doneiconwline
+                                    }
+                                  ${
+                                      isShortScreen
```

Comment:
It's not super clear to me what these style differences are. We should solve them either with media queries (either manually, or more ideally with tailwind) instead of reading the window width in JS.

Reply:
I agree a CSS media query would work well. The CSS is complicated. In a nutshell, this CSS draws the dashed line in a way that works across a variety of screen sizes and browsers.